src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -181,15 +180,13 @@ private void configureBindings() {
 
     drive.setDefaultCommand(drive.drive(x, y, omega));
 
-    led.setDefaultCommand(led.setLEDTheme(LEDTheme.ALLIANCE));
+    led.setDefaultCommand(led.alliance());
```

Comment:
The default command should be a color when there is no note in the robot . You can do this for when the robot is disabled instead.